### PR TITLE
Splitting tabs up

### PR DESCRIPTION
**Pull request by matthewshaver**
[Preview](https://docs-getdbt-com-git-split-tables-dbt-labs.vercel.app/docs/core/connect-data-platform/bigquery-setup#oauth-token-based)

## What are you changing in this pull request and why?

Splitting up tabs into separate examples to simplify discovery. See feedback on https://github.com/dbt-labs/docs.getdbt.com/pull/3121

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

**Review by nghi-ly — COMMENTED** [review]
looks good! since you're editing this page, suggest updating the placeholder text to follow the style guide guidelines https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md#Placeholder-text

**Review by dbeatty10 — APPROVED** [review]
This looks awesome @matthewshaver !

Makes it much easier to see the different options (and hyperlink to them) without the tabs.

File: website/docs/docs/core/connect-data-platform/bigquery-setup.md [hunk 1]
```diff
@@ -74,10 +74,10 @@ my-bigquery-db:
     dev:
       type: bigquery
       method: oauth
-      project: [GCP project id]
-      dataset: [the name of your dbt dataset] # You can also use "schema" here
-      threads: [1 or more]
-      [<optional_config>](#optional-configurations): <value>
+      project: GCP_PROJECT_ID
+      dataset: DBT_DATASET_NAME # You can also use "schema" here
+      threads: 4 #Value must be of 1 or greater
```
Comment (matthewshaver) [inline website/docs/docs/core/connect-data-platform/bigquery-setup.md]:
```suggestion
      threads: 4 # Must be a value of 1 or greater 
```

File: website/docs/docs/core/connect-data-platform/bigquery-setup.md [hunk 2]
```diff
@@ -189,10 +177,10 @@ my-bigquery-db:
...
       type: bigquery
       method: service-account-json
-      project: [GCP project id]
-      dataset: [the name of your dbt dataset]
-      threads: [1 or more]
-      [<optional_config>](#optional-configurations): <value>
+      project: GCP_PROJECT_ID
+      dataset: DBT_DATASET_NAME
+      threads: 4 # Must be a value of 1 or greater
+      [OPTIONAL_CONFIG](#optional-configurations): <value>
```
Comment (nghi-ly) [inline website/docs/docs/core/connect-data-platform/bigquery-setup.md]:
```suggestion
      [OPTIONAL_CONFIG](#optional-configurations): VALUE
```

File: website/docs/docs/core/connect-data-platform/bigquery-setup.md [hunk 3]
```diff
@@ -161,11 +149,11 @@ my-bigquery-db:
...
-      project: [GCP project id]
-      dataset: [the name of your dbt dataset]
-      threads: [1 or more]
-      keyfile: [/path/to/bigquery/keyfile.json]
-      [<optional_config>](#optional-configurations): <value>
+      project: GCP_PROJECT_ID
+      dataset: DBT_DATASET_NAME
+      threads: 4 # Must be a value of 1 or greater
+      keyfile: /PATH/TO/BIGQUERY/keyfile.json
+      [OPTIONAL_CONFIG](#optional-configurations): <value>
```
Comment (nghi-ly) [inline website/docs/docs/core/connect-data-platform/bigquery-setup.md]:
```suggestion
      [OPTIONAL_CONFIG](#optional-configurations): VALUE
```

File: website/docs/docs/core/connect-data-platform/bigquery-setup.md [hunk 4]
```diff
@@ -110,21 +103,19 @@ my-bigquery-db:
...
-      client_id: [client id]
-      client_secret: [client secret]
-      token_uri: [redirect URI]
-      [<optional_config>](#optional-configurations): <value>
+      project: GCP_PROJECT_ID
+      dataset: DBT_DATASET_NAME # You can also use "schema" here
+      threads: 4 # Must be a value of 1 or greater
+      refresh_token: TOKEN
+      client_id: CLIENT_ID
+      client_secret: CLIENT_SECRETS
```
Comment (nghi-ly) [inline website/docs/docs/core/connect-data-platform/bigquery-setup.md]:
```suggestion
      client_secret: CLIENT_SECRET
```

**Review by nghi-ly — COMMENTED** [review]
thank you for this update! you even did extra credit by handling placeholders for additional sections 🔥

**Pull request merged**